test/e2e/elasticsearch_test.go

```diff
@@ -26,6 +27,9 @@ type ElasticSearchTestSuite struct {
 	suite.Suite
 }
 
+var trueVar = true
+var falseVar = false
```

Comment:
You only need to use a single variable - initialised to false when doing the initial deployment and then updated to true when doing the update.